ccmlib/scylla_docker_cluster.py

```diff
@@ -44,6 +46,15 @@ def _update_config(self):
                 'docker_image': docker_image
             }, f)
 
+    def get_node_ip(self, nodeid):
+        try:
+            node = self.nodelist()[nodeid - 1]
+            return node.address()
+        except IndexError:
```

Comment:
Error is a very expensive thing, it is better to change it to the following code:
```
if 0<= nodeid - 1 < len(self.nodelist()):
   return node.address()
 nodeid += len(self.nodelist())
return self.get_ipformat() % nodeid
```